packages/icons/src/library/translation.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * WordPress dependencies
+ */
+import { SVG, Path } from '@wordpress/primitives';
+
+const translation = (
```

Comment:
Should we call this "language" instead of "translation"?

Reply:
Sure, makes sense 👍 
Initially, I called the icon `translation` because that was the name of the [dashicon](https://developer.wordpress.org/resource/dashicons/#translation), but now that it's a different icon it makes more sense to use a different name.
Pushed the commit.